src/Visualization/Python/GenerateXdmf.py

```diff
@@ -21,13 +21,20 @@ def generate_xdmf(file_prefix, output, subfile_name, start_time, stop_time,
                         "python 2, which is deprecated. GenerateXdmf.py might "
                         "hang or run very slowly using python 2. Please use "
                         "python 3 instead.")
+    if subfile_name is None:
+        raise ValueError("The --subfile-name option must be specified.")
```

Comment:
Instead of this error message, add `required=True` to the `--subfile-name` option when it is parsed, line 254 in old diff. It was an oversight on my part to not add that flag when I added the option.